browserify.js

```diff
@@ -1,8 +1,14 @@
-var THREE = require('three');
-var work = require('webworkify');
-var PhysijsWorker = work(require('./physijs_worker.js'));
+class work {
+	constructor(){
+		const work = require('webworkify');
+		return work;
+	}
+}
 
-// inject Three.js and Physijs's Worker
-var Physijs = require('./physi.js')(THREE, PhysijsWorker);
+const THREE = require('three'),
```

Comment:
This shouldn't have been moved around. You also shouldn't just change the syntax from ES5 to ES6+ without any considerations -- this is actually specifically using ES5 so it matches `Physijs` (which is quite old) and so that we don't need to transpile a handful of LoC that are going to be used in a bundler anyway (this package is specifically meant to be used in one). There is no transpilation done in this project itself currently.